blockbench_to_minetest.sed

```diff
@@ -0,0 +1,18 @@
+# Requires sed -r; use sed -i -r -f 'blockbench_to_minetest.sed' 'model.obj' for easiest results
+
+# Remove comments
+/^#\s.*$/d
```

Comment:
```suggestion
/^#.*$/d
```

I don't think comments need to start with a space

Reply:
True, that's redundant, I suppose I put it in there because blockbench's generated comments always have one